Add setup docs (#21)
2324-4bhif-syp/2324-4bhif-syp-project-franklyn: asciidocs/docs/setup.adoc

WantedBy=multi-user.target
----

Make sure to create the service file in the correct directory or move it afterwards with:
[source,shell]
----
sudo mv <path-to-franklyn.service> /etc/systemd/system/
----

Finally, start the service and make sure it's running:
[source, shell]
----
sudo systemctl enable --now franklyn.service
sudo systemctl status franklyn.service
----

=== Student Client
The last thing to do is preparing the student client (also known as openbox). Why is this needed? Per default the generated openbox artifact on GitHub points at our development domain `franklyn.ddns.net`. To work with other servers this domain has to be changed and the jar rebuilt.

To do this clone the repository on a local computer:
[source, shell]
----
git clone https://github.com/2324-4bhif-syp/2324-4bhif-syp-project-franklyn.git
cd 2324-4bhif-syp-project-franklyn
----

Open `openbox/src/main/resources/application.properties` and change:
[source, properties]
----
%prod.websocket.url=wss://franklyn.ddns.net/api
%prod.http.url=https://franklyn.ddns.net/api
----
to
[source, properties]
----
%prod.websocket.url=wss://<your-domain-name>/api
%prod.http.url=https://<your-domain-name>/api
----

> For building the student-client `java 17` or newer is required. Make sure java as well as maven are installed and in the project root run:

Reviewer: ```suggestion
For building the student-client `java 17` or newer and maven is required. With these installed you can redirect to the project root and run:
```
